it/v1/endpoints/CreatePeriodSummaryControllerISpec.scala

```diff
@@ -193,7 +193,7 @@ class CreatePeriodSummaryControllerISpec extends IntegrationBaseSpec {
           AuditStub.audit()
           AuthStub.authorised()
           MtdIdLookupStub.ninoFound(nino)
-          DownstreamStub.onSuccess(DownstreamStub.POST, downstreamUri, OK, desResponse)
+          DownstreamStub.onSuccess(DownstreamStub.POST, downstreamUri, CREATED, desResponse)
```

Comment:
I know it is not part of this ticket, but could you rename `desResponse` to `downstreamResponse` or wtte, as it is quite confusing here